Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -308,7 +347,7 @@ function GetMailboxProp {
         $Prop
     )
 
-    Write-Verbose "GetMailboxProp: [$Prop]: Searching for:[$PassedCN]..."
+    Write-Debug "GetMailboxProp: [$Prop]: Searching for:[$PassedCN]..."
```

Comment:
Do you want to keep this as a Debug?

Reply:
Yes, way too much information on Verbose.